### PR TITLE
Add French translations

### PR DESCRIPTION
**Pull request by sportebois**
Hello

After discovering the Hedy introduction yesterday, I found the approach so nice for children that I wanted to make it available for children around here too. 

This PR adds French translations (the Hedy grammar is still in English though, like it is for other languages too)

A second commit also fixes a few typos I spot in the English texts.

Hope this helps, thank you for building this 🙏

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -29,6 +29,7 @@
     'en': '🇺🇸',
     'nl': '🇳🇱',
     'es': '🇪🇸',
+    'fr': 'ғʀ',
```
Comment (MecuSorin) [inline app.py:32]: Can you give me a link, or a link on how to type that symbol?

Comment (sportebois) [reply app.py:32]: I typed it, and to tell the truth, it was the most difficult part of the translation.
I used an online font converter (probably this one if I remember correctly https://manytools.org/facebook-twitter/smallcaps-text-tool/ ) to get the text to small caps. 

I’m not sure it’s exactly the intended formatting @Felienne expected, but when I run Hedy, the dropdown menu with the languages looks fine with this, so I guess it’s correct.

Comment (Felienne) [reply app.py:32]: Yeah it all works fine, that a lot, it just misses the flag, but I can add that myself!

**Comment by Felienne** [comment]
btw, what I do is I google "French flag emoji" and then I copy it into the code :)

**Pull request merged**